diagnostic.md

```diff
@@ -12,17 +12,19 @@ Graces-MacBook-Pro:wdi grace$ pwd
 Graces-MacBook-Pro:wdi grace$ mkdir try
 Graces-MacBook-Pro:wdi grace$ cd try
 Graces-MacBook-Pro:try grace$
-```
+
 
 What is the full path to the current directory?
 
 _Replace this text with your answer._
+/Users/grace/wdi
```

Comment:
Nope, we went inside the `try` directory. The path is now `/Users/grace/wdi/try`